### PR TITLE
Add ajo classification field group

### PR DESCRIPTION
**Pull request by shagreel**
This classification data will be used in conjunction with the correlationID that was added to the proposition interaction. See https://github.com/adobe/xdm/pull/1453

### CODE REVIEW

File: extensions/adobe/experience/journeyOrchestration/journeyOrchestrationClassification.schema.json [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+  "type": "object",
+  "meta:status": "stable",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": [
+    "https://ns.adobe.com/experience/journeyOrchestration/stepEvents/journey"
+  ],
+  "definitions": {
+    "classification": {
+      "type": "object",
```
Comment (mjasrotia) [inline extensions/adobe/experience/journeyOrchestration/journeyOrchestrationClassification.schema.json:21]: This type: object is not required here as its a definition

Comment (shagreel) [reply extensions/adobe/experience/journeyOrchestration/journeyOrchestrationClassification.schema.json]: removed.

File: extensions/adobe/experience/journeyOrchestration/journeyOrchestrationClassification.example.3.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  "xdm:channel": {
+    "xdm:_id": "Mobile Push",
+    "xdm:_type": "https://ns.adobe.com/xdm/channel-types/push"
+  },
+  "xdm:campaign": {
+    "xdm:campaignID": "19d3b8d6-32fa-4b06-b5fe-2692f828072b",
+    "xdm:campaignName": "My Push Campaign",
+    "xdm:campaignVersionID": "7b203379-1d61-42ef-a096-aded7f957d62"
+  }
+}
```
Comment (mjasrotia) [inline extensions/adobe/experience/journeyOrchestration/journeyOrchestrationClassification.example.3.json:12]: Please change the namespaces in your example files as shown below:

{
  "https://ns.adobe.com/experience/journeyOrchestration/classification/messageID": "",
  "(https://ns.adobe.com/experience/journeyOrchestration/classification/channel": {
    "xdm:_id": "Mobile Push",
    "xdm:_type": "https://ns.adobe.com/xdm/channel-types/push"
  },
 "https://ns.adobe.com/experience/journeyOrchestration/classification/campaign": {
    "xdm:campaignID": "19d3b8d6-32fa-4b06-b5fe-2692f828072b",
    "https://ns.adobe.com/experience/journeyOrchestration/classification/campaignName": "My Push Campaign",
    "https://ns.adobe.com/experience/journeyOrchestration/classification/campaignVersionID": "7b203379-1d61-42ef-a096-aded7f957d62"
  }
}`

Comment (shagreel) [reply extensions/adobe/experience/journeyOrchestration/journeyOrchestrationClassification.example.3.json:12]: Thank you. Fixed.

**Review by mjasrotia — APPROVED** [review]

**Pull request merged**